### PR TITLE
CI: must-add-tests check: use GH label, not text

### PR DESCRIPTION
**Pull request by edsantiago**
Old way: edit commit message, add magic string, re-push

New way: repo maintainer adds a Github label to PR, hits Rerun

I've looked and looked for the history behind this script
and why I didn't do it this way in the first place. I've
concluded that I just never thought of it.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
Curious if the sekrit detector will trigger

**Comment by rhatdan** [comment]
LGTM
Thanks.

**Review by Luap99 — CHANGES_REQUESTED** [review]
I like it overall.
However this is missing a CONTRIBUTING.md update as it still lists adding `[NO NEW TESTS NEEDED]`

**Review by Luap99 — APPROVED** [review]
LGTM
@containers/podman-maintainers PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

Maybe we can combine the two approaches?  The old magic string could lead to auto-apply the label.

**Comment by edsantiago** [comment]
>Maybe we can combine the two approaches? The old magic string could lead to auto-apply the label.

@vrothberg my understanding, based on discussion in another PR, is that there is some value to requiring repo-maintainer signoff on PRs without tests. At a very minimum it will bubble up to someone's attention, and that someone can suggest ways to test. And hence not require the label.

**Pull request merged**
